### DISCUSSION TITLE
Small mover move decision

### DISCUSSION THREAD

**Discussion by NooraAz** [RosettaCommons/rosetta#109] [RosettaScripts] [answered]
When one uses a `Small` mover, which code is used for deciding the magnitude of change in each step? (I would appreciate a path to it)
I mainly need to know if the torsion angle is taken from a **normal** or **uniform** distribution. (or something else?)

**Comment by roccomoretti** [comment, marked as answer]
Should  be in [src/protocols/simple_moves/BackboneMover.hh](https://github.com/RosettaCommons/rosetta/blob/main/source/src/protocols/simple_moves/BackboneMover.hh#L260)

(Looks to be [uniform](https://github.com/RosettaCommons/rosetta/blob/main/source/src/protocols/simple_moves/BackboneMover.cc#L555), which makes sense given that you specify a range versus a center and width.)

**Reply by NooraAz** [reply]
Thanks!
